api-reference/overview.mdx

```diff
@@ -3,18 +3,19 @@ title: "API Overview"
 description: "How to use the Arroyo API"
 ---
 
-In Arroyo 0.4, we've released the initial endpoints of REST API, supporting
-creating and managing pipelines and jobs. The REST API by default runs at
-http://localhost:8000/api.
+As of Arroyo 0.6,
+all functionality of Arroyo is available through the REST API.
+The REST API by default runs at http://localhost:8000/api.
 
-More endpoints are coming. If you're looking to get started with the API, please
+If you're looking to get started with the API, please
```

Comment:
I'm not sure we still need this language now that it's done and documented

Reply:
removed it